diff --git a/include/nativehelper/jni.h b/include/nativehelper/jni.h
index 1c2fb0c..040c4e6 100644
--- a/include/nativehelper/jni.h
+++ b/include/nativehelper/jni.h
@@ -30,7 +30,7 @@
 /* Primitive types that match up with Java equivalents. */
 typedef uint8_t  jboolean; /* unsigned 8 bits */
 typedef int8_t   jbyte;    /* signed 8 bits */
-typedef uint16_t jchar;    /* unsigned 16 bits */
+typedef wchar_t jchar;    /* <-- modified */
 typedef int16_t  jshort;   /* signed 16 bits */
 typedef int32_t  jint;     /* signed 32 bits */
 typedef int64_t  jlong;    /* signed 64 bits */
