diff --git a/crypto/mem_clr.c b/crypto/mem_clr.c
index add1f78..44b00cb 100644
--- a/crypto/mem_clr.c
+++ b/crypto/mem_clr.c
@@ -60,7 +60,7 @@
 #include <openssl/crypto.h>
 
 unsigned char cleanse_ctr = 0;
-
+/*
 void OPENSSL_cleanse(void *ptr, size_t len)
 	{
 	unsigned char *p = ptr;
@@ -75,3 +75,4 @@ void OPENSSL_cleanse(void *ptr, size_t len)
 		ctr += (63 + (size_t)p);
 	cleanse_ctr = (unsigned char)ctr;
 	}
+*/
\ No newline at end of file
